File: WebAPI/Models/Order.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace WebAPI.Models
{
    public class Order : BaseEntity
    {
        [Required]
        public int UserId { get; set; }
        [Required]
        public int DelivererId { get; set; }
        [Required]
        public string HomeAddress { get; set; }
        [Required]
        public string Comment { get; set; }
        [Required]
        public string Status { get; set; }
        [Required]
        public int OrderPrice { get; set; }
        [Required]
        public DateTime DeliveryTime { get; set; }
        public ICollection<OrderProduct> OrderProducts { get; set; }
    }
}

File: Frontend/src/app/model/order.ts
export class Order{
  userId!:number;
  homeAddress!:string;
  comment!:string;
  orderPrice!:number;
  products!: {productId : number, quantity: number} [];
}

export class OrderResponseDto{
  id!: number;
  userId!:number;
  delivererId!:number;
  homeAddress!:string;
  comment!:string;
  orderPrice!:number;
  status!:string;
  deliveryTime!: string;
  deliveryTime2!: number;
  products!: {ingredient:string, name : string, quantity: number} [];
}
